lua/nvim-lsp-setup/utils.lua

```diff
@@ -31,16 +31,16 @@ function M.default_mappings(bufnr, mappings)
 end
 
 function M.disable_formatting(client)
-    client.resolved_capabilities.document_formatting = false
-    client.resolved_capabilities.document_range_formatting = false
+    client.server.document_formatting = false
```

Comment:
shoudl this be server_capabilities ?

Reply:
sorry, I have resolved this spelling errors.